lib/parser/current.rb

```diff
@@ -110,6 +110,15 @@ def warn_syntax_deviation(feature, version)
     require 'parser/ruby32'
     CurrentRuby = Ruby32
 
+  when /^3\.3\./
+    current_version = '3.3.0'
```

Comment:
Should be `3.3.0-dev`, we'll change it to `3.3.0` once it's released (in a year)

Reply:
Cool! Fixed.